src/test/java/edu/cornell/kfs/module/purap/document/service/impl/CuCreditMemoServiceImplTest.java

```diff
@@ -99,16 +95,28 @@ public void setUp() throws Exception {
 
         GlobalVariables.setUserSession(mo14Session);
 	}
+    
+    private DocumentService buildMockDocumentService() {
+        DocumentService mockDocumentService = Mockito.mock(DocumentService.class);
+        Mockito.when(mockDocumentService.createNoteFromDocument(Mockito.any(Document.class), Mockito.anyString())).then(this::buildDocumentNote);
+        return mockDocumentService;
+    }
+    private Note buildDocumentNote(InvocationOnMock invocation) {
+        Document document = invocation.getArgument(0);
+        String noteText = invocation.getArgument(1);
+        
+        Note note = Mockito.mock(Note.class);
+        Mockito.when(note.getNotePostedTimestamp()).thenReturn(dateTimeService.getCurrentTimestamp());
+        Mockito.when(note.getVersionNumber()).thenReturn(Long.valueOf(1L));
+        Mockito.when(note.getNoteText()).thenReturn(noteText);
+        Mockito.when(note.getNoteTypeCode()).thenReturn("TST");
```

Comment:
Is this the correct note type to use? Or does this test not validate the note type very strictly?

Reply:
Note type isn't validated strictly

Reply:
Ok, thanks for confirming.